readers/combined.cc

```diff
@@ -484,25 +486,28 @@ future<mutation_reader_merger::needs_merge> mutation_reader_merger::prepare_one(
             // end are out of data for good for the current range.
             _halted_readers.push_back(rk);
         } else if (_fwd_mr == mutation_reader::forwarding::no) {
-            _to_remove.splice(_to_remove.end(), _all_readers, rk.reader);
-            if (_to_remove.size() >= 4) {
-                auto to_remove = std::move(_to_remove);
-                to_close = parallel_for_each(to_remove, [] (flat_mutation_reader_v2& r) {
-                    return r.close();
+                // FIXME: indentation
```

Comment:
What about fixing the indentation following this patch?

Reply:
I'm okay with that as a follow-up.